__tests__/app.test.js

```diff
@@ -58,26 +58,67 @@ describe("Test /api/articles/:article_id", () => {
         expect(body.votes).toBe(100);
       });
   });
-  test("the method blocks SQL injections", () => {
+  test("ERROR 400:the method blocks SQL injections", () => {
     return request(app).get("/api/articles/1; DROP DATABASE").expect(400);
   });
 
-  test("GET 400: returns an error when the id does not match", () => {
+  test("ERROR 400: returns an error when the id does not match", () => {
     return request(app)
       .get("/api/articles/90")
       .expect(400)
       .then(({ body }) => {
         expect(body.msg).toBe("Bad Request");
       });
   });
-  test("the method blocks an invalid id", () => {
+  test("ERROR 400: the method blocks an invalid id", () => {
     return request(app)
       .get("/api/articles/an-invalid_id")
       .expect(400)
       .then(({ body }) => {
         expect(body.msg).toBe("Bad Request");
       });
   });
+  test("PATCH 202: returns the edited article", () => {
```

Comment:
we will want a 200 for this, 202 means accepted but not complete whereas we will be responding with a completed patched article